Source/WebCore/css/CSSStyleRule.cpp

```diff
@@ -159,8 +159,11 @@ String CSSStyleRule::cssText() const
 
 void CSSStyleRule::cssTextForRules(StringBuilder& rules) const
 {
-    for (unsigned index = 0; index < length(); index++)
-        rules.append("\n  "_s, item(index)->cssText());
+    for (unsigned index = 0; index < length(); index++) {
+        auto ruleText = item(index)->cssText();
+        if (!ruleText.isEmpty())
+            rules.append("\n  "_s, WTFMove(ruleText));
```

Comment:
The `WTFMove()` doesn't hurt here but also doesn't do anything at the moment I believe. StringBuilder doesn't move the String in at the moment as far as I know:
```
void append(const String&);
```

Reply:
Indeed, but it's more future-proof if someone take care of implementing `StringBuilder::append(String&&)` in the future

Reply:
Yes, I don't mind it. I was commenting because Tim asked me whether it helps here.